etc/helm/pachyderm/templates/pachd/config-secret.yaml

```diff
@@ -132,6 +132,9 @@ stringData:
       {{- if .Values.console.enabled}}
       trusted_peers:
       - {{ .Values.console.config.oauthClientID | quote }}
+      {{- range .Values.oidc.additionalClients }}
+      - {{ .id | quote }}
```

Comment:
> Determine if user should be allowed to specify whether an additional client is also a trusted peer of the pachd client


As this is a WIP, would love some feedback about this^ particular issue. Should each additional client be automatically added as a trusted peer to the pachd client? If not, should we require the user to opt-in or opt-out of making an additional client a trusted peer?
For example, we could add the following field to the `additionalClients` schema:
```yaml
- id: my-additional-client
  name: my-additional-client
  secret: ...
  redirect_uris:
    - ...
  trusted_by_pachd: true # should the default be true or false?

Reply:
Good question. Since pachd is the only system minting access tokens, I think that it's fair to assume at this point that all systems that are configured to authenticate against it will want to permission to request pachyderm access tokens. In which case what you have proposed so far is great. 

What do you think @chainlink ?